src/hb-ot-math-table.hh

```diff
@@ -295,6 +388,18 @@ struct MathKern
 
 struct MathKernInfoRecord
 {
+  MathKernInfoRecord* copy (hb_serialize_context_t *c, const void *base) const
+  {
+    TRACE_SERIALIZE (this);
+    auto *out = c->embed (this);
+    if (unlikely (!out)) return_trace (nullptr);
+
+    for (unsigned i = 0; i < 4; i++)
```

Comment:
Use `for` loop iteration / `zip`, or at least `ARRAY_LENGTH` instead of hard-coding 4.